src/Stratis.Bitcoin/Consensus/Validators/BlockValidator.cs

```diff
@@ -57,9 +55,8 @@ public interface IIntegrityValidator
         /// It should be performed even if the block is behind last checkpoint or part of assume valid chain.
         /// TODO specify what exceptions are thrown (add throws xmldoc)
         /// </remarks>
-        /// <param name="block">The block that is going to be validated.</param>
-        /// <param name="chainedHeader">The chained header of the block that will be validated.</param>
-        void VerifyBlockIntegrity(Block block, ChainedHeader chainedHeader);
+        /// <param name="chainedHeaderBlock">The block and chained header that are going to be validated.</param>
+        ValidationContext VerifyBlockIntegrity(ChainedHeaderBlock chainedHeaderBlock);
```

Comment:
Returns comment missing

Reply:
fixed

Reply:
Ah I see what you mean now about returning the context, but this is ok in my opinion